File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/view/nowplaying/INowPlayingFileProvider.java
package com.lasthopesoftware.bluewater.client.playback.view.nowplaying;

import com.lasthopesoftware.bluewater.client.browsing.items.media.files.ServiceFile;
import com.namehillsoftware.handoff.promises.Promise;

public interface INowPlayingFileProvider {
	Promise<ServiceFile> getNowPlayingFile();
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/view/nowplaying/INowPlayingFileProvider.kt
package com.lasthopesoftware.bluewater.client.playback.view.nowplaying

import com.lasthopesoftware.bluewater.client.browsing.items.media.files.ServiceFile
import com.namehillsoftware.handoff.promises.Promise

interface INowPlayingFileProvider {
	val nowPlayingFile: Promise<ServiceFile>
}
